controllers/reconcilers/token/token_manager.go

```diff
@@ -188,10 +189,6 @@ func assignFromSecret(ctx context.Context, c client.Client, cr *v1.Observability
 		index.Gateway = string(targetSecret.Data[ObservatoriumSecretKeyGateway])
 	}
 
-	if index.Tenant == "" {
```

Comment:
do we need to remove this too? We need to make sure that the token refresher still works if the proxy is not used.

Reply:
Yeah you're right I can leave it in. Thanks